guides/v2.2/cloud/project/new-relic.md

```diff
@@ -137,27 +156,103 @@ Using the New Relic dashboard, you can immediately track and find:
+-  **Most time consuming**—Determine time consumption by tracking requests in parallel. For example, you may have the highest transaction time spent in product and category views. If a customer account page suddenly ranks very high in time consumption, your application might be affected by a call or query dragging performance.
+
 -  **Highest throughput**—Identify pages hit the most based on the size and frequency of bytes transmitted.
 
 All collected data details the time spent on an action transmitting data, queries, or _Redis_ data. If queries cause issues, New Relic provides information to track and respond to it.
 
-To learn more about using and customizing New Relic services for your project, see [New Relic Infrastructure documentation](https://docs.newrelic.com/docs/infrastructure/new-relic-infrastructure/getting-started/introduction-new-relic-infrastructure)
+{:.bs-callout-info}
+You can learn more about using the New Relic APM and Infrastructure agents to collect and analyze application data from the [New Relic APM](https://docs.newrelic.com/docs/apm/new-relic-apm/getting-started/introduction-new-relic-apm) and [New Relic Infrastructure](https://docs.newrelic.com/docs/infrastructure/new-relic-infrastructure/getting-started/introduction-new-relic-infrastructure) documentation.
+
+## Monitor performance with alert policies
+
+The New Relic service for {{ site.data.var.ece }} Pro Production environments includes a set of Adobe-generated alert policies to track the following performance metrics:
+
+-  [Apdex score](https://docs.newrelic.com/docs/apm/new-relic-apm/apdex/apdex-measure-user-satisfaction)
+-  error rate
+-  disk space on your Production environment
+
+These policies set thresholds for warning and critical conditions that affect performance based on industry best practices. When your site experiences infrastructure or application issues that trigger an alert threshold, New Relic sends alert notifications so you can proactively address the issue. To use these policies, you must configure notification channels to receive the alert messages.
+
+{:.bs-callout-info}
+For Pro Staging and Integration environments and Starter environments, use [Health notifications]({{ page.baseurl }}/cloud/integrations/health-notifications.html) to monitor disk space.
+
+**Prerequisites:**
+
+-  Credentials to log in to the [New Relic account](#manage-your-new-relic-account) for your Cloud project.
+-  Verify that your Cloud environment is [connected to New Relic](#connect-to-new-relic).
+
+### Review Adobe-generated alert policies
+
+1. Use your Project Owner New Relic credentials to [log in to New Relic](https://login.newrelic.com/login).
+
+1. From the navigation menu, select **Alerts** > **Alert Policies**.
+
+1. In the Search Policies field, search for _Adobe Generated Policy_.
```

Comment:
```suggestion
1. In the _Search Policies_ field, search for **Adobe Generated Policy**.
```

This makes me wonder if any mention of "Adobe-generated" should have a hyphen...I know it should, but if Adobe doesn't use it and it's not in the interface, I wonder if it consistent. May have to remove the hyphen everywhere if it is considered the official name. I'm just thinking aloud.

Reply:
The policy name for the generated policies is  "Adobe Generated Policy" -- I was using Adobe-generated policies to refer to the policies generically, but I also wondered. Let's see what technical reviewers prefer.